public/app/features/alerting/unified/components/rule-editor/ConditionField.tsx

```diff
@@ -54,31 +54,32 @@ export const ConditionField: FC = () => {
   }, [condition, expressions, options, setValue]);
 
   return (
-    <Field
-      label="Condition"
-      description="The query or expression that will be alerted on"
-      error={errors.condition?.message}
-      invalid={!!errors.condition?.message}
-    >
-      <InputControl
-        name="condition"
-        render={({ field: { onChange, ref, ...field } }) => (
-          <Select
-            aria-label="Condition"
-            {...field}
-            width={42}
-            options={options}
-            onChange={(v: SelectableValue) => onChange(v?.value ?? null)}
-            noOptionsMessage="No queries defined"
-          />
+    <Card>
+      <Card.Heading>Set alert condition</Card.Heading>
+      <Card.Meta>Select one of your queries or expressions set above that contains your alert condition.</Card.Meta>
+      <Card.Actions>
+        {options.length && (
+          <Field error={errors.condition?.message} invalid={!!errors.condition?.message}>
+            <InputControl
+              name="condition"
+              render={({ field: { onChange, ref, ...field } }) => (
+                <RadioButtonList options={options} onChange={onChange} {...field} />
+              )}
+              rules={{
+                required: {
+                  value: true,
+                  message: 'Please select the condition to alert on',
+                },
+              }}
+            />
+          </Field>
+        )}
+        {!options.length && (
+          <Alert title="Neither queries nor expressions have been configured" severity="warning">
```

Comment:
I think it's probably easier without double negation :)

```suggestion
          <Alert title="No queries or expressions have been configured" severity="warning">
```